extension/opampextension/factory.go

```diff
@@ -27,6 +27,7 @@ func createDefaultConfig() component.Config {
 		Server: &OpAMPServer{},
 		Capabilities: Capabilities{
 			ReportsEffectiveConfig: true,
+			ReportsHealth:          true,
```

Comment:
setting the `ReportsHealth` capability to `true` for now, but please let me know if we should keep this disabled by default for now

Reply:
I agree we should have this enabled by default, as it should allow us to remove the requirement for the healthcheck extension in the Collector.